internal/locate/region_cache.go

```diff
@@ -2198,6 +2136,30 @@ func (c *RegionCache) cacheGC() {
 				}
 				c.mu.Unlock()
 			}
+
+			// Check remaining regions and update sync flags
+			for _, region := range remaining {
+				syncFlags := region.getSyncFlags()
+				if syncFlags&needAsyncReloadReady > 0 {
+					// the region will be reload soon on access
+					continue
+				}
+				if syncFlags&needAsyncReloadPending > 0 {
+					region.setSyncFlags(needAsyncReloadReady)
```

Comment:
May I understand the flag change here is to avoid too frequent meaningless reloading?

Reply:
Yes, it tries to limit the rate of reloading here.